Objects/typeobject.c

```diff
@@ -9011,8 +9045,8 @@ super_descr_get(PyObject *self, PyObject *obj, PyObject *type)
     }
 }
 
-static int
-super_init_without_args(InterpreterFrame *cframe, PyCodeObject *co,
+int
+_PySuper_GetTypeArgs(InterpreterFrame *cframe, PyCodeObject *co,
                         PyTypeObject **type_p, PyObject **obj_p)
```

Comment:
```suggestion
                     PyTypeObject **type_p, PyObject **obj_p)
```

The line was aligned with an opening parenthesis of a parameter list.